pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -208,6 +208,21 @@ func (sna *SnapshotAlpha) CreateFromSource(ctx context.Context, source *Source,
 	if !waitForReady {
 		return nil
 	}
+	if source.Driver == "csi-vxflexos.dellemc.com" {
+		us, err := sna.dynCli.Resource(v1alpha1.VolSnapGVR).Namespace(namespace).Get(ctx, snap.GetName(), metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		status, ok := us.Object["status"].(map[string]interface{})
+		if !ok {
+			errors.Errorf("Failed to convert status to map, Volumesnapshot: %s, Status: %v", snap.GetName(), status)
+		}
+		status["readyToUse"] = true
+		us.Object["status"] = status
+		if _, err := sna.dynCli.Resource(v1alpha1.VolSnapGVR).Namespace(namespace).UpdateStatus(ctx, us, metav1.UpdateOptions{}); err != nil {
```

Comment:
Can we use `Patch` here?

Reply:
I dont think you can patch a status. That why UpdateStatus does a PUT internally to replace the whole object

Reply:
Got it - sounds good.